### ISSUE TITLE
Left/Right Balance Fields missing in Raw Data table

### ISSUE THREAD

**Issue by FW2017** [GoldenCheetah/GoldenCheetah#4067]
Raw Data table in Activities > Details does not show left/right balance fields despite them being present in FIT file. 

![image](https://user-images.githubusercontent.com/94026303/141053817-6c7a562d-d43e-49bc-b60d-f2061b8a646a.png)
![image](https://user-images.githubusercontent.com/94026303/141053908-311f2f39-9838-4af3-9a78-776afed4d812.png)

**Comment by amtriathlon** [comment]
Some apps/devices include an L/R Balance field incorrectly encoded according to the FIT specification, which requires the right bit being set:

![image](https://user-images.githubusercontent.com/1444784/103826072-e7b7f600-5054-11eb-80da-7e68dc85aec0.png)

GoldenCheetah maps unknown to NA. You can check independently if this is the case for your FIT file import it in Garmin Connect and see if left-right balance is displayed.

**Comment by grmpl** [comment]
This does not make sense: Ignoring a value which is just "unknown", but not undefined and making a working software buggy just because of a unclear specification.

**Comment by amtriathlon** [comment]
The specification seems clear enough to me, anyway if you want to discuss the specification https://www.thisisant.com/ seems the place to do it.

**Comment by liversedge** [comment]
The undefined value means just that. The ANT spec is clear and the reasoning behind it is likely to support transmission and connection issues not an extension of the standard on the whim of a manufacturer.
If we change to support this non-standard use case it will impact users where the standard has been applied correctly and likely mean invalid data as a result.
As Ale has said, Wahoo or ANT should shed some light on this, perhaps there is a way of identifying this and handling without breaking parsing for other users.

**Comment by grmpl** [comment]
Sorry, I wasn't aware this software is paid for by GARMIN as ANT is. So you support GARMIN devices only.

**Comment by liversedge** [comment]
The ANT+ and FIT file support is developed by our team and not affiliated with Garmin.
In fact many man hours went in to developing both to provide an open source solution.
If you can show how support for non-conformant devices can be added without breaking support for conformant devices we will happily add support.

**Comment by liversedge** [comment]
Wahoo have been in touch to understand this issue better and opening back up to support any dialog or updates whilst looking into this

**Comment by amtriathlon** [comment]
According to [users reports in the users forum](https://groups.google.com/g/golden-cheetah-users/c/PAX-oRdK_i8/m/6qa5fMTLBQAJ) this issue seems to happen only when the power meter is paired to the Wahoo head unit using ANT+, and it doesn’t when paired using Bluetooth LE.

**Comment by amtriathlon** [comment]
Wahoo has fixed this issue in the firmware version released today: [ELEMNT WF48-16044 - 20 June, 2023](https://support.wahoofitness.com/hc/en-us/articles/217285397-ELEMNT-Firmware-Release-Notes)
*Fixed: FIT file does not contain L/R bit for balance data with certain power meters

**Comment by Cyclingscott** [comment]
Hi, I realise that this is an older thread but I have been having the same issue with a Garmin head unit for a while now. I have tried both an older edge 830 and a new Edge 1040 and both seem to have the same problem, i.e. Left-Right balance doesn't display in Golden Cheetah but does display on the Garmin Connect site. Is there a fix available for this issue? I have downloaded the latest GC version and both head units are up to date. 

Thanks in advance for any help, Scott
